contracts/HATVaultsRegistry.sol

```diff
@@ -493,11 +295,11 @@ contract HATVaultsRegistry is Ownable {
         if (swapData.amount == 0) revert AmountToSwapIsZero();
         IERC20 _HAT = HAT;
         (swapData.hatsReceived, swapData.amountUnused) = _swapTokenForHAT(IERC20(_asset), swapData.amount, _amountOutMinimum, _routingContract, _routingPayload);
-
         governanceHatReward[_asset] = swapData.amountUnused * governanceHatReward[_asset] / swapData.amount;
 
         for (uint256 i = 0; i < _beneficiaries.length; i++) {
             uint256 hackerReward = swapData.hatsReceived * hackersHatReward[_asset][_beneficiaries[i]] / swapData.amount;
+            uint256 hackerAmountSwapped = (swapData.amount - swapData.amountUnused) * hackersHatReward[_asset][_beneficiaries[i]] / swapData.amount;
```

Comment:
is this not simply swapData.amount - swapData.amountUnused?

Reply:
yes but per hacker

Reply:
because each event is per hacker. and we're still missing one for the gov swap. i will do it in a different pr

Reply:
ah ok sorry my confusino